scanner.go

```diff
@@ -28,28 +28,40 @@ import (
 // See other methods for custom usage
 
 var (
-	LF       byte   = '\n'
-	CR       byte   = '\r'
-	newLines []byte = []byte{CR, LF}
+	// LF -> linefeed
+	LF byte = '\n'
+	// CR -> carriage return
+	CR       byte = '\r'
+	newLines      = []byte{CR, LF}
 )
 
+// Scanner conatins all stuff for reading a buffered file
 type Scanner struct {
 	f       *os.File      // The file provided by the client.
 	r       *bufio.Reader // Buffered reader on given file.
 	keepnls bool          // Keep the newline sequence in returned strings
 	token   []byte        // Last token returned by split (scan).
 	err     error         // Sticky error.
+	seekers []seeker      // Internal use
```

Comment:
When a field  (or func/method) starts with a capital letter, it means that the field is public. With a lower case letter, it means that the field is private.